### ISSUE TITLE
Module not found: Can't resolve 'fs'

### ISSUE THREAD

**Issue by princevora** [thanathip41/tspace-mysql#4]
Having Module not found: Can't resolve 'fs' in your repo i have faced the similar issue before that was solved automatically.
Even the fs is currently installed in my package this is second time i had this error 

![image](https://github.com/thanathip41/tspace-mysql/assets/122656064/d772a8c1-6ad8-4a9b-aab9-95aa1a44809b)

**Comment by thanathip41** [comment]
Sorry for the slow reply. Could you please tell me about your version of Next.js?

Thank you

**Comment by princevora** [comment]
its v14.2.3. I think its client side error

**Comment by thanathip41** [comment]
> its v14.2.3. I think its client side error

Yes, maybe can you fix the config in Nextjs ? , i'm trying to find a solution to fix that in my library soon.

Thank you

**Comment by princevora** [comment]
You can just prevent the thing to run but thats not a solution, when you import a Model in client components i see these errors

**Comment by thanathip41** [comment]
Sorry for the late response.

You can fix this by updating your `next.config.js`:

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  experimental: {
    serverComponentsExternalPackages: ['tspace-mysql'],
  },
}

module.exports = nextConfig
```

Thank you
